mender-deb-package

```diff
@@ -78,7 +71,8 @@ cat << EOF > debian/rules
 	dh \$@
 
 override_dh_auto_build:
-	env GOOS=linux GOARCH=arm GOARM=5 CGO_ENABLED=1 make build
+	env CGO_CFLAGS="-I${LIBLZMA_INSTALL_PATH}/include" CGO_LDFLAGS="-L${LIBLZMA_INSTALL_PATH}/lib" \
+        GOOS=linux GOARCH=arm GOARM=6 CGO_ENABLED=1 make build
```

Comment:
Don't you need: CC=$CC \

Reply:
CC It is already in the environment, so here is just passed to make. Not strictly needed (neither in mender-convert recipe)

Reply:
I think in general it's recommended to drop `env` and put `make` first instead, because variables in the environment don't override variables in the makefile, but variables given directly to make do. In this case @apodogrocki would be right: You do need `CC=$CC`.

Reply:
Thanks. I understand now :)

I will rework

Reply:
Done :+1: